packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -560,15 +451,14 @@ test.serial('writeGranules() does not persist records to Dynamo or Postgres if D
     granuleId,
   } = t.context;
 
-  const fakeGranuleModel = {
-    storeGranuleFromCumulusMessage: () => {
-      throw new Error('Granules dynamo error');
-    },
-    describeGranuleExecution: () => Promise.resolve({}),
+  const fakeGranuleModel = new Granule();
```

Comment:
Nope, I added that so that some of the other methods were available to the test. 
Looking closer, I can add just a method for generateGranule that returns a valid granule. 
742e99e361